### ISSUE TITLE
system and server status: commands do not execute

### ISSUE THREAD

**Issue by jpgpi250** [webmin/webmin#1759]
I have 3 servers with webmin, all v2.001, installed, using
```
version="2.001"
sudo wget http://nav.dl.sourceforge.net/project/webadmin/webmin/${version}/webmin_${version}_all.deb
sudo dpkg --install webmin_${version}_all.deb
```

Two of the servers execute the command (If monitor goes down, run command) perfectly, the result is as expected.
The third server doesn't appear to be executing any commands.
I've tried several commands on all servers, 2 work / one doesn't
When I execute the command in a terminal, the result is as expected on ALL 3 servers, which leads me to believe it's a webmin problem on that specific server.

Any ideas to solve this problem?

Thanks for your time and effort.

![image](https://user-images.githubusercontent.com/24669386/196418915-4f072b59-faa0-4716-84a8-2c0dc79ae7c0.png)

expected result: an entry in syslog, works on 2 servers. Works on all 3 servers if executed from command line.

**Comment by iliajie** [comment]
Also check if `System and Server Status: Scheduled Monitoring` (a blue button <img width="137" alt="image" src="https://user-images.githubusercontent.com/4426533/196495910-b883e310-28c7-4e9c-81aa-0c239605c919.png">) is setup correctly.

**Comment by jpgpi250** [comment]
@iliaross All of the above settings on a working webmin match the settings on the not working webmin

**Comment by jpgpi250** [comment]
found the cause. for some reason, the crontab job '/etc/webmin/status/monitor.pl', scheduled to run every 5 minutes (per scheduled monitoring setting') executed only once (after webmin started) and then never again. I noticed this by looking at the 'status history' of the individual monitors. I checked 'sudo crontab -e', nothing wrong there, i than executed 'sudo /etc/webmin/status/monitor.pl' from the command line once. For some reason (unknown), the crontab job then started working on the regular interval (5 minutes) and everything worked.
